pages/DSNP/Types/GraphChange.md

```diff
@@ -32,48 +30,24 @@ Different change types have different meanings.
 | 0 | Unfollow | Remove a Follow relationship |
 | 1 | Follow | Create a Follow relationship |
 
-### createdAt
-
-- MUST be set to the milliseconds since Unix epoch at time of signing
-- MUST be unique for the given `fromId` and `objectId` pair
-
 ### fromId
 
 - MUST be a [DSNP User Id](../Identifiers.md#dsnp-user-id)
-- MUST be the [signer](../Signatures.md) of the Announcement
+- MUST directly or via a chain of delegation have authorized the creation of the Announcement
 
 ### objectId
 
 - MUST be a [DSNP User Id](../Identifiers.md#dsnp-user-id)
 
-### signature
-
-- MUST be an [Announcement Signature](../Signatures.md) over all fields except the signature field
-
 ## Non-Normative
 
-### Replay Attacks
-
-Clients must ignore any Graph Change event that comes after another event with the same signature.
```

Comment:
I don't see how this is related to DIP-145.

Reply:
@shannonwells it is unclear. Here's why I did it in this DIP:

1. Replay attacks are only possible with announcement level signatures.
2. created_at was only added for replay attack prevention.

So once we take away signatures, neither of these have a reason to exist either.